lang/cs/Org.Apache.REEF.IMRU.Tests/ImruDriverCancelTests.cs

```diff
@@ -198,8 +198,8 @@ private SystemStateMachine GetDriverState(object driver)
                 .BindNamedParameter(typeof(InvokeGC),
                     jobDefinition.InvokeGarbageCollectorAfterIteration.ToString(CultureInfo.InvariantCulture))
                 .Build();
-                
-            return driverConfig;
+
+            return driverConfigBuilder;
```

Comment:
Now that we've reverted to the old way of handling config, we should revert the change in this file. The only thing it does is variable rename, which is not correct anymore (it's `driverConfig` again, since we're building it immediately)